src/editor/ui_menus/main_menu_map_options.cc

```diff
@@ -394,10 +394,16 @@ MainMenuMapOptions::MainMenuMapOptions(EditorInteractive& parent, Registry& regi
 
 	tags_box_.add_space(labelh_);
 
-	tags_box_.add(new UI::Textarea(&tags_box_, UI::PanelStyle::kWui, UI::FontStyle::kWuiLabel, 0, 0,
-	                               max_w_, labelh_, _("Waterway length limit:")));
+	UI::Textarea* ww_text =
+	   new UI::Textarea(&tags_box_, UI::PanelStyle::kWui, UI::FontStyle::kWuiLabel, 0, 0, max_w_,
+	   labelh_, _("Ferry range:"));
+	std::string ww_tooltip = _("Set maximum length of waterways for ferries. If ferry range is set "
+	   "to 0, then the map will not allow building ferry yards and waterways (ferry routes).");
```

Comment:
The UI already shows "Deactivated" for a ferry range <2. So the last sentence can be skipped imho.

What about: "Allow ferries and ferry yards on this map by defining a maximum range for Ferries. The value reflects the amount of allowed fields."